src/krate.rs

```diff
@@ -696,6 +696,17 @@ pub fn index(req: &mut Request) -> CargoResult<Response> {
         ));
     }
 
+    if vec![params.get("user_id"), params.get("team_id")]
+        .iter()
+        .any(|s| s.is_some())
+    {
+        let not_yanked_versions = sql::<Bool>(
+            "crates.id = ANY (SELECT vs.crate_id FROM versions vs WHERE vs.crate_id = crates.id AND vs.yanked IS FALSE)",
```

Comment:
Should this just be `EXISTS (SELECT 1 FROM versions WHERE versions.crate_id = crates.id AND versions.yanked IS FALSE)`?